src/components/show-products/styles.css

```diff
@@ -4,7 +4,8 @@
 
 /* Customize the label (the container) */
 .checkbox-label {
-  display: block;
+  display: flex;
+  justify-content: space-between;
   position: relative;
   cursor: pointer;
   user-select: none;
```

Comment:
I didn't know about this `user-select:none` but [it seems it doesnt work on Chrome](https://developer.mozilla.org/en-US/docs/Web/CSS/user-select) :eyes:

Reply:
Maybe, a better idea could be that you can use [the "native" properties](https://html.spec.whatwg.org/multipage/input.html#the-input-element) in an input checkbox: `checked` & `disabled` (overall last one). After this PR will be merged because you have a bug reported (#27) where a team might work with the solution. As always it's only an idea.


![image](https://user-images.githubusercontent.com/8689897/141538719-e02eb69a-1dfc-4b10-b8cd-cd3a6167568d.png)